styles.css

```diff
@@ -1 +1,751 @@
+}
+
+h3,
+h3.program {
+  margin-top: 30px;
+}
+
+img.event1,
+img.event2 {
+  width: 100%;
+  height: 0%;
+  margin: 20px 0;
+}
+
+section.partners {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 20px 20px;
+  background-color: #424146;
+}
+
+img.partner1,
+img.partner2,
+img.partner3,
+img.partner4,
+img.partner5 {
+  width: 15%;
+}
+
+div.partners-logos {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  padding: 20px 0;
+}
+
+footer {
+  display: flex;
+  flex-direction: row;
+  background-color: white;
+  align-items: center;
+  justify-content: space-around;
+  height: 200px;
+  overflow: hidden;
+}
+
+div.footer-logo {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 20px 0;
+}
+
+p.copyright {
+  font-size: 24px;
+  color: black;
+  padding: 20px 0;
+}
+
+p.copyright-about {
+  font-size: 24px;
+  color: white;
+  padding: 20px 0;
+}
+
+div.social-links {
+  background-color: #231f20;
+  display: flex;
+  flex-direction: row;
+  justify-content: end;
+  padding: 5px;
+}
+
+a.social {
+  color: white;
+  padding-right: 60px;
+}
+
+.partners-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-items: center;
+  gap: 5px;
+}
+
+section.about-section {
+  background-image: url("./Images/background-opacity.png");
+  background-size: cover;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 20px 20px;
+  background-color: white;
+}
+
+section.past-graduates {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background-color: white;
+}
+
+p.about-us-para {
+  margin: auto;
+  margin-top: 5%;
+  line-height: 1.5;
+  background-color: white;
+  padding: 5%;
+}
+
+h1.about-us-title {
+  font-size: 2.5rem;
+  color: #ec5242;
+  text-align: center;
+}
+
+img.my-aboutus-logo {
+  width: 30%;
+  margin: 25px auto;
+  border: 1px solid #d3d3d3;
+  padding: 10px;
+}
+
+div.graduates-images {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 70%;
+  margin: 30px auto;
+  margin-bottom: 15px;
+  object-fit: contain;
+}
+
+img.graduates-image {
+  width: 30%;
+  height: 0%;
+  margin: 20px 0;
+  border: 1px solid #d3d3d3;
+  padding: 10px;
+}
+
+footer.about-footer {
+  background-color: #231f20;
+}
+
+p.past-graduates-para {
+  margin: 20px;
+}
+
+.about-mylogo {
+  margin: 20px;
+  text-align: center;
+}
+
+a.my-email {
+  color: #ec5242;
+}
+
+@media only screen and (min-width: 768px) {
+  ul.menu {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    width: 100%;
+    height: 100px;
+    overflow: hidden;
+    justify-content: space-between;
+  }
+
+  nav {
+    background-color: white;
+    overflow: hidden;
+  }
```

Comment:
Great job @waseemqureshi2020 🤝

- [ ] Kindly add padding to this nav bar so that the items can move away from the border. You may also add a height so that it doesn't appear so huge as related to the design provided. Kindly fix this nav bar for both the home and about sections.
- Your design
![second nav](https://user-images.githubusercontent.com/63917725/197341227-88d141e7-c811-47ef-b6a1-0c654d39a8d3.PNG)
- Design Provided
![design-nav](https://user-images.githubusercontent.com/63917725/197341298-009a886d-58ae-4f90-a794-4d1bf21fbc51.PNG)

Reply:
ok, I have followed your instruction and now the nav bar is similar to the design provided.